src/Compilers/Core/Portable/Operations/UnaryOperatorKind.cs

```diff
@@ -1,24 +1,21 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System.Collections.Immutable;
-
 namespace Microsoft.CodeAnalysis.Semantics
 {
-    public enum SimpleUnaryOperationKind
+    /// <summary>
+    /// Kind of unary operator
+    /// </summary>
+    public enum UnaryOperatorKind
     {
         None = 0x0,
 
         BitwiseNegation = 0x1,
         LogicalNot = 0x2,
-        PostfixIncrement = 0x3,
-        PostfixDecrement = 0x4,
-        PrefixIncrement = 0x5,
-        PrefixDecrement = 0x6,
-        Plus = 0x7,
-        Minus = 0x8,
-        True = 0x9,
-        False = 0xa,
-        BitwiseOrLogicalNot = 0xb,
+        Plus = 0x3,
+        Minus = 0x4,
+        True = 0x5,
+        False = 0x6,
+        BitwiseOrLogicalNot = 0x7,
```

Comment:
Thanks, you are correct. It seems C# has separate operator kinds for BitwiseNegation and LogicalNot. VB has single Not operator to represent LogicalNot (bool operand), BitwiseNegation (integral operand) and BitwiseOrLogicalNot (object operand)

---
In reply to: [134530885](https://github.com/dotnet/roslyn/pull/21407#discussion_r134530885) [](ancestors = 134530885)